src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -486,4 +496,14 @@ protected function getTemplate(
 
         return $template;
     }
+
+    public function deprecateTemplate($oldTemplateName, $newTemplateName)
+    {
+        @trigger_error(sprintf(
+            'The "%s" template is deprecated.'
+            .' Use "@SonataAdminBundle/CRUD/Association/%s" instead.',
```

Comment:
why is this part `@SonataAdminBundle/CRUD/Association/` hardcoded?

Reply:
my bad, forgot to remove it when I was moving it from other PR, will be fixed.